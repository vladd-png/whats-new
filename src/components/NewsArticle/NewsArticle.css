img {
  width: 500px;
  height: 250px;
  object-fit: cover;
}

.imgContainer {
}

.newsCard {
  width: 500px;
  background-color: white;
  margin: 10px;
  height: 500px;
  display: flex;
  justify-content: space-between;
  flex-direction: column;
}

.newsDescription {
  /* margin-bottom: 80px; */
}

.newsHeadline {
  color: black;
  font-size: 1.2em;
  margin-top: -20px;
}

.footerInfo {
  background-color: #B46AF1;
  width: 500px;
  font-size: 20px;
}

.arrowImg {
  height: auto;
  width: 80px;
}

.urlLink {
  justify-content: space-around;
  display: flex;
  align-items: center;
  text-decoration: none;
  color: white;
}

.newsDescription {
  color: black;
}
