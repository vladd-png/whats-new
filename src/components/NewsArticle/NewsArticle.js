import React from 'react';
import './NewsArticle.css';

// NEWSARTICLE COMPONENT CODE GOES HERE
//code to make a commit

export default NewsArticle;
