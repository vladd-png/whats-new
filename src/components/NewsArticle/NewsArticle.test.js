import React from 'react';
import { shallow } from 'enzyme';
import NewsArticle from './NewsArticle';

describe('NewsArticle', () => {

  it('should be an instance of NewsArticle component', () => {
    const wrapper = shallow(<NewsArticle
        props={[
          {headline: 'Spiderman Two', body:'Spidermans A Fraud', img:'url.link', link:'url.link'},
          {headline: 'Giant Chicken', body: 'Sits at Denver Museum', img:'url.link', link:'url.link'}
        ]}
      />);
      expect(wrapper).toMatchSnapshot();
  });

});
