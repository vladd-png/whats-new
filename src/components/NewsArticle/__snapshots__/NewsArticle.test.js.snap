// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should be an instance of NewsArticle component 1`] = `
<section
  className="newsCard"
>
  <section
    className="imgContainer"
  >
    <img />
  </section>
  <h2
    className="newsHeadline"
  />
  <p
    className="newsDescription"
  />
  <footer
    className="footerInfo"
  >
    <a
      className="urlLink"
    >
      Link To Article
      <img
        alt="arrow icon"
        className="arrowImg"
        src="arrow.png"
      />
    </a>
  </footer>
</section>
`;
